pennylane_lightning_gpu/_serialize.py

```diff
@@ -119,26 +173,17 @@ def _serialize_sparsehamiltonian(ob, wires_map: dict, use_csingle: bool):
     return sparsehamiltonian_obs(data, indices, offsets, wires)
 
 
-def _serialize_hermitian(ob, wires_map: dict, use_csingle: bool):
-    if use_csingle:
-        rtype = np.float32
-        hermitian_obs = HermitianObsGPU_C64
-    else:
-        rtype = np.float64
-        hermitian_obs = HermitianObsGPU_C128
+def _serialize_hermitian(ob, wires_map: dict, use_csingle: bool, use_mpi: bool):
+    hermitian_obs, rtype = _hermitian_ob_dtype(use_csingle, use_mpi)
```

Comment:
```suggestion
def _serialize_hermitian(ob, use_csingle: bool, use_mpi: bool):
    hermitian_obs, rtype = _hermitian_ob_dtype(use_csingle, use_mpi)
```

Reply:
As discussed, let's keep the current interface as it was. We can change it later and update it for `lightning`, `kokkos `and `L-GPU`  together.